src/Microsoft.DotNet.Wpf/tests/UnitTests/Directory.Build.targets

```diff
@@ -0,0 +1,36 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../'))" />
+
+  <ItemGroup>
```

Comment:
This section is needed for tests that require PresentationNative:
https://github.com/dotnet/wpf/pull/9471/files#diff-29b181700610c640b4a8cd2b9cd25af0e791ed8b90ec215f8343cab55d0de8a8R5-R26